### PR TITLE
add browse instructions to indexer in docker-compose.yml

### LINKED ISSUE #63: Print what to do next message before indexer service exits
At the moment a `docker-compose --env-file getuigenverhalen.env up` will bring up the user interface the log ends with

```
indexer_1        | Indexing snippets...
indexer_1        | Indexing documents...
indexer_1        | Indexing done.
workingtitleclosereader_indexer_1 exited with code 0
```

It would be nice if it would say where to see the user interface http://localhost:8080.

### PR DESCRIPTION
**Pull request by cwmeijer**
Fixes #63 by adding a print command to the index part in the docker-compose.yml. Another option would have been to put the print in the indexer python script but that did not seem in place at that point.
Not completely happy with the long line in the docker-compose in the current solution. I can't find a way to split this line though as sh seems to want only a single argument to execute.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -24,11 +24,7 @@ services:
...
     depends_on:
       - "elasticsearch"
-    command: ["/utils/wait-for-it.sh", "elasticsearch:9200",
-      "--timeout=0",
-      "--", "/utils/index.py",
-      "data/vocabulary_ids.txt",
-      "data/vocabulary_bow.txt",
-      "elasticsearch"]
+    command: ["sh","-c",
+"/utils/wait-for-it.sh elasticsearch:9200 --timeout=0 -- /utils/index.py data/vocabulary_ids.txt data/vocabulary_bow.txt elasticsearch && echo Now browse to http://localhost:8080/ for the user interface."]
```
Comment (jspaaks) [inline docker-compose.yml]: I prefer to break this up into an array with its elements on different lines, it's easier to diff that way.

**Review by jspaaks — CHANGES_REQUESTED** [review]

**Comment by sverhoeven** [comment]
You can chop up long lines in yaml with '>' see https://yaml-multiline.info/

File: docker-compose.yml (hunk 1 above)
Comment (jspaaks) [reply docker-compose.yml]: Never mind, I just noticed that dowsnt work, apparently

Comment (sverhoeven) [inline docker-compose.yml]: Maybe this works:
```suggestion
    command: 
      - "sh"
      - "-c"
      - > 
        /utils/wait-for-it.sh elasticsearch:9200 --timeout=0 -- 
        /utils/index.py data/vocabulary_ids.txt data/vocabulary_bow.txt elasticsearch
        && 
        echo Now browse to http://localhost:8080/ for the user interface.
```

Comment (cwmeijer) [reply docker-compose.yml]: I saw this suggestion too late. I committed something similar now. Let me know if you prefer yours, then I can change it.

**Review by sverhoeven — APPROVED** [review]
I have no preference between "\n\n" or ">". what is there now is also easy to read and diff.

**Review by jspaaks — APPROVED** [review]

**Comment by fdiblen** [comment]
@cwmeijer What is needed to close this PR?

**Comment by jspaaks** [comment]
dont wanna wait any more

**Pull request merged**